src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/__tests__/components/message/ErrorReport.test.tsx

```diff
@@ -37,7 +37,7 @@ describe('components > ErrorReport.tsx', () => {
     }
     const initialState = getInitialStateMock({ formValidations: mockValidationState});
     const component = mountComponent(initialState);
-    const genericErrorText = initialState.language.language.form_filler.error_report_description;
+    const genericErrorText = initialState.language.language.form_filler['error_report_description'];
```

Comment:
Just curious, whats the reason for this change? Linting complaints or something else?

Reply:
Typescript error, we used to have an 'any' as type for the language prop, this is now defiend as
```ts
export interface ILanguage {
    [key: string]: string | ILanguage;
}
```
Which then forces the fix. Couldn't figure out how to do this differently, but please enlighten me if you have a solution 🤓

Reply:
That does seem correct to me, so not sure why it needs this change... Anyway, it should work as expected, I think the upside of getting rid of `any` is worth this. We can dig into this later if needed. 😁